### DISCUSSION TITLE
Jekyll_app1htmlFile/jekyll/onefileapp/2020/09/09/Aplikacje_html.1

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Jekyll_app1htmlFile#15] [Komentarze]
# Jekyll_app1htmlFile/jekyll/onefileapp/2020/09/09/Aplikacje_html.1

Aplikacje w postaci pliku HTML 1. QR Code - generator i czytnik off-line (o1) 2. Edytor Markdown (marked.js) + MathJax (i1) 3. Zaawansowane zmiany w tekście - wyrażenia regularne (o1) 4. Edytor wizualny WYSIWYG html w przeglądarce (o1)

http://localhost:4000/Jekyll_app1htmlFile/jekyll/onefileapp/2020/09/09/Aplikacje_html.1.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
